src/Book.java

```diff
@@ -0,0 +1,72 @@
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.*;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class Book {
+    private final String bookName;
+    private final String file;
+
+    public Book(String bookName) {
+        this.bookName = bookName;
+        String filePath = "D:/IT Hillel/JavaPro/lesson14/src";
```

Comment:
it is better to use relative path rather  than an absolute path, in your case, app will work only on your machine